### ISSUE TITLE
No method for initialize value Datepicker

### ISSUE THREAD

**Issue by mrtrinhvn** [luyadev/luya#1972]
### What steps will reproduce the problem?
In the model: 
public function ngRestAttributeTypes()
    {
        return [  
            'created_timestamp' => ['class' => Date::class, 'emptyMessage' => '-', 'format' => 'dd/MM/yyyy'],
        ];
    }
There is no option for innit the default value
The format always is dd.MM.yyyy
datepicker twoo small on mobile device for selecting date
### What is the expected result?
There is an option for innit the default value, eg current date
format' => 'dd/MM/yyyy' take effection
On mobile device, the datepicker display enough clear to be selected

### What do you get instead? (A Screenshot can help us a lot!)

![z1634844832411_1fca7cf9799f2b35cae307e99d55c359](https://user-images.githubusercontent.com/3391289/69623384-60d4d380-1075-11ea-8ecc-18ed2615ac56.jpg)

### Additional infos

| Q                | A
| ---------------- | ---
| LUYA Version     | 
| PHP Version      |
| Platform         | Apache/XAMPP/MAMPP/etc.
| Operating system | Windows/Linux Server/OSX/etc.

**Comment by nadar** [comment]
But the init value is not related to format date i assume?

**Comment by mrtrinhvn** [comment]
Yes, value is not related to format, but the format have no effect. The date always is dd.MM.yyy
So, by the way, I tell about some issues of the datepicker

**Comment by nadar** [comment]
So what is then the actually problem you like to address? The formatting?
